docs/deployment/deployPDMultiRegion.md

```diff
@@ -0,0 +1,195 @@
+
+An example of the YAML configuration for these environment variables:
+
+```yaml
+K8S_STATEFUL_SET_NAME=pingdirectory
+K8S_STATEFUL_SET_SERVICE_NAME=pingdirectory
+
+K8S_CLUSTERS=us-east-2 eu-west-1
+K8S_CLUSTER=us-east-2
+K8S_SEED_CLUSTER=us-east-2
+K8S_NUM_REPLICAS=3
+
+K8S_POD_HOSTNAME_PREFIX=pd-
+K8S_POD_HOSTNAME_SUFFIX=.us-cluster.ping-devops.com
+K8S_SEED_HOSTNAME_SUFFIX=.us-cluster.ping-devops.com
+
+K8S_INCREMENT_PORTS=true
+LDAPS_PORT=8600
+REPLICATION_PORT=8700
+```
+
+These environment variable settings would map out like this:
+
+| Seed | Instance | Host name | LDAP | REPL |
+| :---: | --- | --- | :---: | :---: |
+|      | CLUSTER: us-east-2
+| ***  | pingdirectory-0.us-east-2 | pd-0.us-cluster.ping-devops.com | 8600  | 8700 |
+|      | pingdirectory-1.us-east-2 | pd-1.us-cluster.ping-devops.com | 8601  | 8701 |
+|      | pingdirectory-2.us-east-2 | pd-2.us-cluster.ping-devops.com | 8602  | 8702 |
+|      | CLUSTER: eu-west-1
+|      | pingdirectory-0.eu-west-1 | pd-0.eu-cluster.ping-devops.com | 8600  | 8700 |
+|      | pingdirectory-1.eu-west-1 | pd-1.eu-cluster.ping-devops.com | 8601  | 8701 |
+|      | pingdirectory-2.eu-west-1 | pd-2.eu-cluster.ping-devops.com | 8602  | 8702 |
+
+## Cluster Startup Walkthrough
+
+Yes, that was a _ton_ of variable conversation.
+This is done to for flexibility to accommodate various infrastructure constraints. For example, in some environments you cannot use the same port for each instance, so we must accommodate incrementing ports.
+
+Next, it's helpful to know what happens when a cluster starts, to understand why the initial creation of a cluster must be very prescriptive.
+
+1. The first pod must start on it's own and become healthy. This is critical to prevent replication islands. The very first time the very first pod starts, we call it "GENESIS". All other pods are dependent on this `SEED_POD` in the `SEED_CLUSTER` starting correctly on it's own. The entire purpose of defining `SEED_POD` and `SEED_CLUSTER` variables is avoid multiple genesis scenarios.
+
+2. once the first pod is healthy, it begins dns querying combinations of hostnames at their LDAPS port to find another Directory instance.
+
+In our first cluster, this would be the hostname of pingdirectory-1. but it could also be pingdirectory-0 of another cluster. Once the query returns successful, creation of the replication topology automatically begins.
+
+From this point onward, the order in which instances start is less important.
+
+## Deploy the Helm Example
+
+Clone the `getting-started` [Repository](https://github.com/pingidentity/pingidentity-devops-getting-started) to get the Helm values yaml for the exercise (30-helm/multi-region/pingdirectory), then:
+
+1. Modify any external hostnames in the sample values yaml files as necessary - see the lines under `## CHANGEME` comments
+
+2. Deploy the first set of pods
+
+    ```sh
+    kubectx west
+    helm upgrade --install west pingidentity/ping-devops -f 01-west.yaml
```

Comment:
release name should be `sample` for both as it relates to:
`K8S_POD_HOSTNAME_PREFIX: "sample-pingdirectory-"`

Reply:
Good catch